metricbeat/tests/test.sh

```diff
@@ -1,32 +1,31 @@
 #!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh core/php/7.2.8/20181108151533
+#/
 
-TESTDIR="$(dirname "${0}")"
-PLANDIR="$(dirname "${TESTDIR}")"
-SKIPBUILD=${SKIPBUILD:-0}
+set -euo pipefail
 
-hab pkg install --binlink core/bats
+if [[ -z "${1:-}" ]]; then
+  grep '^#/' < "${0}" | cut -c4-
+	exit 1
+fi
 
+TEST_PKG_IDENT="${1}"
+export TEST_PKG_IDENT
+hab pkg install core/bats --binlink
 hab pkg install core/busybox-static
 hab pkg binlink core/busybox-static ps
 hab pkg binlink core/busybox-static wc
+hab pkg install "${TEST_PKG_IDENT}"
 
-source "${PLANDIR}/plan.sh"
-
-if [ "${SKIPBUILD}" -eq 0 ]; then
-    # Unload the service if its already loaded.
-    hab svc unload "${HAB_ORIGIN}/${pkg_name}"
+hab svc load "${TEST_PKG_IDENT}"
```

Comment:
My tests fail unless I ensure the supervisor is running before this ``hab svc load``.  You could try something like the following as in the [journalbeat pr](https://github.com/habitat-sh/core-plans/pull/2601 ):

```diff
--- a/metricbeat/tests/test.sh
+++ b/metricbeat/tests/test.sh
@@ -19,6 +19,9 @@ hab pkg binlink core/busybox-static ps
 hab pkg binlink core/busybox-static wc
 hab pkg install "${TEST_PKG_IDENT}"

+# Ensure supervisor is running
+until hab sup status; do jobs | grep -q "hab sup run" || eval "hab sup run &"; echo "Waiting for supervisor to start..."; sleep 1; done
+
 hab svc load "${TEST_PKG_IDENT}"

 # Allow service start
```